pages/products/flexberry-ember/3.x/components/efd3_object-list-view.en.md

```diff
@@ -0,0 +1,210 @@
+`previousPage`| Method to move to the previous page.
+`gotoPage`| Method to move on a given page.
+`nextPage`| Method to move to the next page.
+`editFormRoute`| Specifies the name of the route edit form, which will open the model.
+`singleColumnHeaderTitle`| Header for mobile view component, instead of column names. If not specified or equal to "" then the header of the table in the component is hidden.
+`colsConfigButton`| Flag (`true`/`false`) enable/disable the display of buttons in the user settings.
+`bottomPagination`| Flag that determines whether to display pagination bottom/up (`true`/`false`).  
+
+Default values:
+
+```javascript
+action: 'rowClick',
+createNewButton: false,
+refreshButton: false,
+defaultSortingButton: true,
+orderable: false,
+rowClickable: true,
+showCheckBoxInRow: false,
+showDeleteButtonInRow: false,
+showDeleteMenuItemInRow: false,
+showEditMenuItemInRow: true,
+colsConfigButton: true,
+bottomPagination: true
+```
+
+## Customize lists
+
+Part of Flexberry Objectlistview control includes a control panel that you can customize and add custom buttons.
+
+Control panel, likes buttons in rows, might be supplemented with custom buttons that implement the necessary functions to work with list. Described in detail in [Settings list](efd3_setting-lists.html).
+
+## Export to Excel
+
+Technology `Ember Flexberry` provides the ability to export lists. To export became available in the app, you need to implement [settings backend](fan_odata-export-to-excel.html).
+
+In the template directly form must be specified
+
+```hbs
+{% raw %}{{flexberry-objectlistview
+...
+exportExcelButton=true
+{% endraw %}
+```
+
+Export settings can be saved by specifying a name. You can also change the name of the exported columns.
+
+## Deleting list items
+
+To remove items from the list method is used `beforeDeleteRecord`. The method supports asynchronous mode, it is possible to return `promise` as the value of the result.
+
+### Deleting all selected items
+
+To configure simultaneous removal of all selected list items you need in the controller to register event `beforeDeleteAllRecords`:
+
+```javascript
+import ListFormController from './list-form';
+
+export default ListFormController.extend({
+  actions: {
+    beforeDeleteAllRecords(modelName, data) {
+      if (modelName === 'application-user') {
+        data.cancel = false;
+      }
+    }
+  }
+});
+```
+
+and specify it in the template list:
+
+```hbs
+{% raw %}
+{{flexberry-objectlistview
+  ...
+  beforeDeleteAllRecords=(action 'beforeDeleteAllRecords')
+  ...
+}}{% endraw %}
+```
+
+## Check list before removing items
+
+When working with the list can return a `promise` using `return` in the handler's body `beforeDeleteRecord`, it is possible to implement asynchronous logic. If you do return < promise > then the delete operation will be called after promise is fulfilled. If he will be terminated, the deletion will fail.
```

Comment:
"If you do return < promise > then the delete operation will be called after promise is fulfilled. If he will be terminated, the deletion will fail." => "If the < promise > is returned then the delete operation is called after promise is fulfilled. If it is terminated, the deletion fails."